core/shared/src/main/scala/fs2/pipe.scala

```diff
@@ -135,6 +136,28 @@ object pipe {
     }
   }
 
+  /** Debounce the stream with a minimum delay of `d` between each element */
+  def debounce[F[_], I](d: FiniteDuration)(implicit F: Effect[F], scheduler: Scheduler, ec: ExecutionContext): Pipe[F, I, I] = {
```

Comment:
Ignore this, it didn't work out the way I was thinking. :+1: for this implementation.